pkg/storage/ec/client.go

```diff
@@ -164,9 +166,17 @@ func (ec *ecClient) Get(ctx context.Context, nodes []*proto.Node, es eestream.Er
 	}
 	rr, err = eestream.Decode(rrs, es, ec.mbm)
 	if err != nil {
+		for _, rr := range rrs {
+			_ = rr.Close()
```

Comment:
I am really confused when we should use `utils.LogClose()` and when to ignore the error. Why is it OK to ignore the error here and on the other places in this PR?

Reply:
My general guidelines for errors / logging are:
1. return errors unless you intend to handle it, never use global logger
2. the only exception is package main (and hacking things together)

The solution here would be to use `utils.CombineErrors` where the first is the actual error that failed everything. The `rr.Close` may provide some other useful information about the situation that lead to the issue.